modules/041-linstor/templates/nodegroupconfiguration-install-kernel-headers.yaml

```diff
@@ -21,27 +21,31 @@ spec:
     # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     # See the License for the specific language governing permissions and
     # limitations under the License.
-    
+
     # DRBD kernel-module-injector requires the kernel sources to be installed.
 
-    # Install actual kernel headers
     version_in_use="$(uname -r)"
     if bb-is-distro-like? centos; then
-      bb-yum-install "kernel-devel-${version_in_use}"
-    fi
-    if bb-is-distro-like? debian; then
-      bb-apt-install "linux-headers-${version_in_use}"
-    fi
+      # Install actual kernel headers
+      #   kernel         --> kernel-devel
+      #   kernel-core    --> kernel-devel
+      #   kernel-lt      --> kernel-lt-devel
+      #   kernel-lt-core --> kernel-lt-devel
+      package_name="$(rpm -qf --qf "%{NAME}-devel" "/lib/modules/${version_in_use}" | sed 's/-core-/-/')"
```

Comment:
```suggestion
      package_name="$(rpm -qf --qf "%{NAME}-devel" "/lib/modules/${version_in_use}" | sed 's/-core//')"
```

Reply:
It's better to have full word matching here, to avoid situations, when kernel name starts with `core`.

Any way, I just updated pattern to match `-core` only in the end of package